public/style.css

```diff
@@ -14,4 +16,55 @@
 
 ul {
   list-style: none;
+  overflow: hidden;
+  justify-content: center;
+}
+
+li {
+  display: inline-block;
+  width: 30%;
+  margin-top: 5%;
```

Comment:
For margin top/bottom I would use `px` or `rem`, percentage based margins can have unexpected results